lib/livebook/utils.ex

```diff
@@ -47,7 +47,7 @@ defmodule Livebook.Utils do
     random_part = :crypto.strong_rand_bytes(9)
     binary = <<node_part::binary, random_part::binary>>
     # 16B + 9B = 25B is suitable for base32 encoding without padding
-    Base.encode32(binary, case: :lower)
+    Livebook.Config.random_boot_id() <> Base.encode32(binary, case: :lower)
```

Comment:
Maybe we could make boot id a part of the binary representation, like 16B node hash + 3B boot bytes + 11B random bytes, wdyt?